spm_funcs.py

```diff
@@ -54,8 +54,17 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    img = nib.load(fname)
+    data = img.get_fdata()
+
```

Comment:
You might consider leaving out the blank lines here - because the code is pretty easy to follow without, and would be more compact.  But not a big deal either way.